includes/functions.php

```diff
@@ -3963,4 +3963,84 @@ function pmpro_get_ip() {
 	$client_ip = preg_replace( '/[^0-9a-fA-F:., ]/', '', $client_ip );
 	
 	return $client_ip;
+}
+
+/**
+ * Determines if this order can be refunded
+ * @param  object $order The order that we want to refund
+ * @return bool Returns a bool value based on if the order can be refunded
+ */
+function pmpro_allowed_refunds( $order ) { 
+
+	if( empty( $order ) || empty( $order->gateway ) ) {
+		return false;
+	}
+
+	/**
+	 * Processes a refund for Stripe orders
+	 *
+	 * @since TBA
+	 *
+	 * @param bool Default return value is false to determine if the refund was successfully processed. 
+	 * @param object $order The Member Order we want to refund
+	 * @return bool If the refund was successfully processed
+	 */
+	
```

Comment:
```suggestion
	/**
	 * Allow filtering the list of statuses that can not be refunded from.
	 *
	 * @since TBD
	 *
	 * @param array $allowed_statuses The list of statuses that can not be refunded from.
	 */
```